CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
+tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+- Started keeping a changelog. Added a changelog file, and added a changelog nagger to CI.
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [1.2.2] - 2024-12-13
+
+### Fixed
+
+- Fixed the galois_group problem.
+
+- Fixed zero-dimensional cone in cones in PolyhedralGeometry.
```

Comment:
I can't tell if "cone" is a regular word here or the name of a command, likewise for "cones". My guess is this is perhaps meant?
```suggestion
- Fixed zero-dimensional cone in `cones` in PolyhedralGeometry.
```
But again that's too vague, and doesn't reference any relevant PR(s)